app/vite.config.js

```diff
@@ -20,8 +13,16 @@ import { searchForWorkspaceRoot } from 'vite';
 import { defineConfig } from 'vitest/config';
 
 const API_PATH = path.join('..', 'api');
+
+/*
+ * @TODO This extension path is hardcoded to the env default (./extensions). This won't work
+ * as expected when extensions are read from a different location locally through the
+ * EXTENSIONS_LOCATION env var
+ */
```

Comment:
Couldn't we use the new `@directus/env` package here? (👀 🚀)

Reply:
I don't know if that matters 🤔 This is really only meant for dev mode when running Directus from the repo. Ideally this is removed completely from vite and instead handled through the API just like in production mode!

Reply:
Yup 👍